mod/profperm.php

```diff
@@ -109,9 +109,9 @@ function profperm_content(App &$a) {
 	}
 
 	$o .= '<div id="prof-update-wrapper">';
-	if($change) 
+	if($change)
```

Comment:
It looks like someone has removed the curly braces but left trailing spaces behind. As you said, you are not going to change CS stuff but only `$a` stuff, this is fine for me.